op-chain-ops/deployer/integration_test/apply_test.go

```diff
@@ -27,6 +27,8 @@ import (
 const TestParams = `
 participants:
   - el_type: geth
+    el_extra_params:
+      - "--gcmode=archive"
```

Comment:
Just curious, why do we need archive mode?

Reply:
Sorry, I was asking why this test specifically needs archive mode. I didn't think it accessed any storage roots for the deployment testing